src/groups/mqb/mqbc/mqbc_incoreclusterstateledger.cpp

```diff
@@ -688,7 +688,7 @@ int IncoreClusterStateLedger::applyRecordInternalImpl(
         }
 
         iter->second.d_ackCount += 1;
-        if (iter->second.d_ackCount == d_ackQuorum) {
+        if (iter->second.d_ackCount == getAckQuorum()) {
```

Comment:
```suggestion
        const unsigned int ackQuorum = getAckQuorum();
        if (iter->second.d_ackCount == ackQuorum) {
```
Can use `ackQuorum` in the `BALL_LOG_INFO` below. This prevents the slight chance of misinformation if quorum changes in between the `if` check and the logging.